editor/editor_node.cpp

```diff
@@ -1789,6 +1794,46 @@ void EditorNode::_edit_current() {
 			if (FileAccess::exists(source_scene + ".import")) {
 				editable_warning = TTR("This scene was imported, so changes to it won't be kept.\nInstancing it or inheriting will allow making changes to it.\nPlease read the documentation relevant to importing scenes to better understand this workflow.");
 			}
+			else if (current_node->is_class("MeshInstance3D")){
```

Comment:
I don't think is the right place to do so. Usually, we put that kind of code in their respective EditorPlugin, definitely not in the editor_node.cpp file.